docs/java-rest/high-level/ml/put-trained-model.asciidoc

```diff
@@ -39,6 +39,8 @@ include-tagged::{doc-tests-file}[{api}-config]
 <5> Optionally, a human-readable description
 <6> Optionally, an object map contain metadata about the model
 <7> Optionally, an array of tags to organize the model
+<8> The default inference config to use with the model. Must match the underlying
```

Comment:
I think the new object needs to be added in the definitions section too (e.g. https://www.elastic.co/guide/en/elasticsearch/reference/master/put-inference.html#ml-put-inference-trained-model)